Source/Blazorise.Bootstrap5/wwwroot/modal.js

```diff
@@ -1,4 +1,4 @@
-import { addClassToBody, removeClassFromBody } from "../Blazorise/utilities.js?v=1.4.2.0";
+import { addClassToBody, removeClassFromBody } from "../Blazorise/utilities.js?v=1.4.2";
```

Comment:
The version must contain 4 values, ending in 0 in this case.

Reply:
Any special reason? I might change the approach anyway, I'm still testing. I just found out that MSBuild supports executing C# code, that might be better because it will be cross platform build-able more easily.

Reply:
Because the version that is sent from Blazorise at runtime must match the version that is used by the JS to load additional JS modules. 


```js
public override string ModuleFileName => $"./_content/Blazorise/closable.js?v={VersionProvider.Version}";
```

And making it always explicitly have 4 values make it easier to handle without introducing additional rules like trimming the end. And find and replace is much easier.

Reply:
I see. Just the way I got the version from : 
![image](https://github.com/Megabit/Blazorise/assets/22283161/b01675de-a25a-47d4-9b22-81e0d694bc26)
Does not yield the 0. But I can figure out some other symbol with it. Surely there's a way to get the assembly version. 
I'll probably just use C# to get the current assembly version. We'll see.
Thanks.